README.md

```diff
@@ -26,13 +26,23 @@ This is a monorepo that houses the following packages. Please refer to the READM
 - [`@metamask/subject-metadata-controller`](packages/subject-metadata-controller)
 - [`@metamask/transaction-controller`](packages/transaction-controller)
 
-Here is a graph that shows the dependencies among all packages:
+Or, in graph form:
 
 ![Dependency graph](assets/dependency-graph.png)
 
 > **Note**
 > To regenerate this graph, run `yarn generate-dependency-graph`.
 
```

Comment:
Intended together with https://github.com/MetaMask/controllers/pull/989#discussion_r1031935330
```suggestion.markdown
Or, in graph form:[^1]

![Dependency graph](assets/dependency-graph.png)

Refer to individual packages for usage instructions.



[^1]: To regenerate this graph, run `yarn generate-dependency-graph`.
```

Reply:
Ah. I keep forgetting GitHub supports footnotes. That will put the note all the way at the bottom of the document. I guess that's okay.